applications/connecting_applications_to_services/understanding-service-binding-operator.adoc

```diff
@@ -0,0 +1,83 @@
+Binding data:: Information about a service that you use to configure the behavior of other resources within the cluster. Examples include credentials, connection details, volume mounts, secrets, and so on.
+Binding connection:: Any connection that establishes an interaction between the connected components such as a bindable backing service and an application requiring that backing service.
+
+
+[id="about-service-binding-operator"]
+== About Service Binding Operator
+{servicebinding-title} consists of a controller and an accompanying Custom Resource Definition (CRD) for service binding. It manages the data plane for workloads and backing services. The Service Binding Controller reads the data made available by the control plane of backing services and projects this data to workloads according to the rules specified through the `ServiceBinding` resource.
+
+As a result, the {servicebinding-title} enables workloads to use backing services or external services by automatically collecting and sharing binding data with the workloads. The process involves making the backing service bindable and binding the workload and the service together.
+
+
+[id="making-an-operator-managed-backing-service-bindable"]
+=== Making an Operator-managed backing service bindable
+To make a service bindable, as an Operator provider you need to expose the binding data required by workloads to bind with the services provided by the Operator. You can provide the binding data either as annotations or as descriptors in the CRD of the Operator that manages the backing service.
+
+
+[id="binding-a-workload-together-with-a-backing-service"]
+=== Binding a workload together with a backing service
+By using the {servicebinding-title}, as an application developer, you need to declare the intent of establishing a binding connection. You must create a `Service Binding` CR  that references the backing service. This action triggers the {servicebinding-title} to project the exposed binding data into the workload. The {servicebinding-title} receives the declared intent and binds the workload together with the backing service.
+
+The CRD of the {servicebinding-title} supports the following APIs:
+
+* *Service Binding* with the `binding.operators.coreos.com` API group.
+* *Service Binding (Spec API Tech Preview)* with the `servicebinding.io` API group.
++
+[IMPORTANT]
+====
+*Service Binding (Spec API Tech Preview)* with the `servicebinding.io` API group is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
```

Comment:
Avoid "recommends". Instead of "Red Hat recommends", direct users to take the recommended action. This allows Red Hat to be more prescriptive in documentation and prevent any user uncertainty, and is easier for upstream/downstream coordinated efforts.

https://redhat-documentation.github.io/supplementary-style-guide/#recommend